applications/deployments/what-deployments-are.adoc

```diff
@@ -12,6 +12,8 @@ The `Deployment` and `DeploymentConfig` API objects in {product-title} provide t
 * `DeploymentConfig` objects involve one or more _replication controllers_, which contain a point-in-time record of the state of a deployment as a pod template. Similarly, `Deployment` objects involve one or more _replica sets_, a successor of replication controllers.
 * One or more pods, which represent an instance of a particular version of an application.
 
+Use `Deployment` objects unless you need a specific feature or behavior provided by `DeploymentConfig` objects.
+
 ////
 Update when converted:
 [role="_additional-resources"]
```

Comment:
Can we update `Building blocks of a deployment` to?

```
Users do not have to manipulate replica sets, replication controllers, or pods owned deployment or deployment config objects.
```

Reply:
Updated - let me know how it looks now

Reply:
+1